cli_tools/gce_vm_image_import/importer/disk_inspection_processor.go

```diff
@@ -41,8 +41,13 @@ func (p *diskInspectionProcessor) process(pd persistentDisk,
 		return pd, err
 	}
 
-	pd.isUEFICompatible = p.args.UefiCompatible || ir.HasEFIPartition
-	loggableBuilder.SetUEFIMetrics(pd.isUEFICompatible, ir.HasEFIPartition)
+	isHybrid := ir.UEFIBootable && ir.BIOSBootable
+	if !p.args.UefiCompatible && isHybrid {
+		log.Printf("This disk can boot with both BIOS and UEFI. It will be imported as BIOS booting. " +
```

Comment:
"It will be imported as BIOS booting" doesn't sound right. Maybe get Shawnique to review this message?